package.json

```diff
@@ -244,7 +244,7 @@
     "unified": "^6.1.6",
     "webpack-bundle-analyzer": "3.3.2"
   },
-  "packageManager": "yarn@3.2.2",
+  "packageManager": "yarn@3.2.3",
```

Comment:
This is supposed to fix [our patch](https://github.com/emotion-js/emotion/tree/f2608b3a542164a3622453f306e2b782929a3020/.yarn/patches) not being correctly applied. Note that the patch itself could be removed if we'd upgrade Jest.